ansible/roles/host-ocp4-installer/tasks/main.yml

```diff
@@ -4,12 +4,26 @@
 - name: Generate install_config.yaml
   import_tasks: generate_install_config.yml
 
-- name: Run the installer
-  become: no
-  tags:
-  - run_installer
-  command: openshift-install create cluster --dir=/home/{{ ansible_user }}/{{ cluster_name }}
-  async: "{{ 2 * 60 * 60 }}"
+- block:
+    - name: Run the installer
```

Comment:
Please maintain consistent YAML indentation within files.

While we don't have an official rule about indent or not for sequences, the `Set up Student User` block later in this file does not indent sequences so additions to this file should match that style.

Reply:
Adapted block style.